hoodie-client/src/test/java/com/uber/hoodie/TestHoodieClient.java

```diff
@@ -350,12 +455,18 @@ public void testDeletes() throws Exception {
     }
 
 
+    //TODO (na) : Fix this test case for COPY_ON_WRITE, AutoRollback doesn't work cleanly
     @Test
     public void testCreateSavepoint() throws Exception {
```

Comment:
Some race conditions after switching to DFS when using rollback for COPY_ON_WRITE, working on finding the root cause, will fix the 2 test cases asap.